fuel-ts-sdk/package.json

```
@@ -35,7 +43,9 @@
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@fuels/connectors": "^0.36.1",
     "@reduxjs/toolkit": "^2.2.7",
+    "fuels": "^0.100.0",
```

Biome (2.1.2)

[error] The key fuels was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)